tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "strict": true,
     "allowSyntheticDefaultImports": false,
     "outDir": "build",
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "noImplicitAny": false
```

Comment:
Instead of changing this I would remove alphabetised imports requirements. It is real pain

Reply:
Or you can set tslint to correct imports on file save, then you don't have to do it manually.

Reply:
I know, but from my experience this is the most irritating thing you can set up in tslint. It does not check anything important (like use of any) what's more not every editor allows to set it up like that (ofc you could bind it to commit hook too).

I'm not saying you have to change this, but seriously guys, don't tell me, failed build because of this didn't trigger you? ;)

@blyszcz @barograf

Reply:
Let's be pragmatic...

Reply:
Alphabetised imports are more annoying than useful IMO, we can change that. Regarding `noImplicitAny` - I guess this change was temporary, it needs to be changed back.